src/cloudai/test_definitions/nemo_launcher.py

```diff
@@ -59,34 +61,52 @@ class TrainingModelData(BaseModel):
     data_prefix: str = "[\"1.0\",'${data_dir}/my-gpt3_00_text_document']"
 
 
+class Optim(BaseModel):
+    """Optimizer configuration."""
+
+    model_config = ConfigDict(extra="forbid")
+    name: str = "fused_adam"
+    bucket_cap_mb: str = "200"
+    contiguous_grad_buffer: str = "True"
+    overlap_grad_sync: str = "True"
+    overlap_param_sync: str = "True"
+
+
 class TrainingModel(BaseModel):
     """Training model configuration."""
 
     model_config = ConfigDict(extra="forbid")
+    activations_checkpoint_num_layers: Literal["null"] = "null"
+    data: TrainingModelData = Field(default_factory=TrainingModelData)
+    fsdp: bool = True
```

Comment:
How does this default values affect the IL-E2E teams existing Nemo Launcher runs?

Reply:
FSDP and other parallelization optimization should ideally not be in the default. 

If we agree it to be default, then the test definitions for IL E2E test needs to be modified to explicitly set these flags as `false`. All the other Nemo test also needs to be defined that way.

Out of the box, without those changes, it will fail to run.  wdyt? 
cc: @srinivas212

Reply:
Correct, other configurations should be updated accordingly. We may need to explicitly unset these values by setting them to ~

Reply:
We need to make the following design choices:
* Are we going to adopt the idea presented in this PR?
  * If yes, should we update existing configurations to unset any arguments? Please note that we will need to repeat this process whenever we add a new argument to the test template. Additionally, the PR unsets arguments by setting their values to `~`. This requires the field to be a string. Should we adopt this approach, or do you have alternative suggestions?
  * If no, how will we provide new arguments?